app/models/admin/delete/confirm.js

```diff
@@ -24,7 +24,18 @@ const DogsRemovedViewModel = (count) => {
   })
 }
 
+const OwnersRemovedViewModel = (count) => {
+  return new ViewModel({
+    breadcrumbs,
+    titleHtml: count === 1 ? '1 dog owner record has been deleted' : `${count} dog owner records have been deleted`,
+    bodyContent: [
+      'Raise a support ticket if you need to recover a deleted owner record.'
```

Comment:
Is there not some wording about 'within the next 90 days'?

Reply:
test/integration/narrow/routes/admin/delete/owners.test.js
app/views/admin/delete/owners-confirm.njk

Reply:
Yes, it's in the nunjucks template - test and template here.